### ISSUE TITLE
Insufficient colour contrast between the three empty elements on the upturned card area

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#28]
Increase the contrast between the three areas. This image shows those three grey areas when empty.

![AccessibleSolitaire_TalkBackCardSelectionState](https://github.com/user-attachments/assets/5228decc-fbea-4bfb-ab4b-158aa81fd4eb)

**Comment by gbarkerz** [comment]
I've replaced the green borders to the empty upturned card piles with black borders. This still may not be as high contrast as I'd like given the width of the border, but I'll close this issue now and review it if necessary based on feedback. The change is in V1.18 but for some reason Google hasn't yet provided an associated accessibility report, so I don't know if Google will still call out an issue with the changed visuals.

![image](https://github.com/user-attachments/assets/f3f2f6b4-e273-45ab-9196-e9c055f2eb9a)
